org.eclipse.m2e.discovery.oss/src/main/resources-filtered/connectors.xml

```diff
@@ -203,6 +203,26 @@
       </overview>
     </catalogItem>
 
+    <!-- collection of various components for M2E 2.0 -->
+    <catalogItem>
+      <categoryId>org.eclipse.m2e.discovery.category.lifecycles</categoryId>
+      <m2e-versions>2.0</m2e-versions>
+      <description>M2E Components for various Maven plug-ins.</description>
+      <groupId>lifecycles</groupId>
+      <id>org.eclipse.m2e.discovery.lifecyclemapping.m2e-components</id>
+      <kind>lifecycles</kind>
+      <license>EPL</license>
+      <name>m2e-components</name>
+      <provider>Community maintained</provider>
+      <p2>
+        <repositoryUrl>https://eclipseguru.github.io/m2eclipse-components/</repositoryUrl>
+        <iuId>org.eclipseguru.m2e.extras.components.feature.feature.group</iuId>
```

Comment:
@fbricon Do you know if `<lifecycleMappingIU>` is needed?

Reply:
yes, it allows m2e to discovery which feature from the catalog holds a particular lifecycleMappingId referenced in a pom file